multimodal/src/autogluon/multimodal/utils/few_shot_learning.py

```diff
@@ -91,11 +91,33 @@ def predict(self, data, as_pandas: Optional[bool] = False):
         features = self.extract_embedding(data)
 
         preds = self.clf.predict(features)
+        # import ipdb
+
+        # ipdb.set_trace()
         if as_pandas:
             preds = self._automm_predictor._as_pandas(data=data, to_be_converted=preds)
         return preds
 
+    def predict_proba(self, data, as_pandas: Optional[bool] = False, as_multiclass: Optional[bool] = True):
+        if not self._fit_called and not self._model_loaded:
+            warnings.warn(
+                "Neither .fit() nor .load() is not invoked. Unexpected predictions may occur. Please consider calling .fit() or .load() before .predict()"
+            )
+
+        features = self.extract_embedding(data)
+        preds = self.clf.decision_function(features)
+        probs = np.exp(preds) / np.sum(np.exp(preds + 1e-8), axis=1, keepdims=True)
```

Comment:
Refer to https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/predictor.py#L2288